src/ampl.js

```diff
@@ -1,33 +1,29 @@
-var Remarkable = require('remarkable');
-var htmlparser = require("htmlparser2");
+import Remarkable from 'remarkable';
+import htmlparser from 'htmlparser2';
 
-var remarkable = new Remarkable('full');
+import { getDims, updateImgTags } from './imageDims.js';
+import { createAmpPage } from './templates.js';
 
-import { getDims } from './imageDims.js';
-import { html2Amp } from './templates.js';
+const remarkable = new Remarkable('full');
 
-export function parse(mdString, css, callback) {
```

Comment:
We all agree the word `function` must be cleansed from all code! :hankey: 
![two legs bad](http://25.media.tumblr.com/tumblr_lmwjx0TXpT1ql36ygo1_400.jpg)